scripts/parse.js

```diff
@@ -5,8 +5,8 @@ import crypto from "crypto";
 
 const MODULES_DIR = path.join("modules");
 const OUTPUT_FILE = path.join(config.dist.directory, "modules.json");
-const ACCEPTED_EXTENSIONS = [".json", ".js", ".ts", ".jsx", ".tsx", ".md", ".py"];
-const META_FILE = ["meta.json"]
+const ACCEPTED_EXTENSIONS = [".json", ".js", ".ts", ".jsx", ".tsx", ".md", ".py", ".png"];
```

Comment:
Adding `.png` here can be an issue, this means that we will be parsing all png image files that the module contains, and we don't support that just yet.